### ISSUE TITLE
No items showing in Elevators

### ISSUE THREAD

**Issue by auddie2015** [Thutmose/Thut#116]
There are no items in the mod for me.

#### What happens:

Nothing at all. There are no items, no recipes, nothing.

#### What you expected to happen:

Items and recipes (not being snarky LOL)

#### Steps to reproduce:

1. It could be a conflict but there is no crash. I don't see anything obvious to reproduce it.
2.
3.
...

____
#### Affected Versions (Do *not* use "latest"): Replace with a list of all mods you have in.

- Thutcore:  thutcore-1.12.2-5.19.7.jar
- Minecraft: 1.12.2
- Forge: 1.12.2-forge1.12.2-14.23.4.2758
![untitled](https://user-images.githubusercontent.com/15572587/44692737-57078f00-aa32-11e8-852c-bb281e25f9e3.png)

**Comment by Thutmose** [comment]
can you include a complete mod list? screenshot of mods folder would work, the export from say a forced crash report would be better (iirc holding f3+c for a few seconds forces a crash)

**Comment by auddie2015** [comment]
LOL Yup, that crashed it. 

[crash-2018-08-27_19.59.50-client.txt](https://github.com/Thutmose/Thut/files/2325998/crash-2018-08-27_19.59.50-client.txt)

Yeah, I prolly have too many mods... But I love them! 

Thanks for checking it for me.

**Comment by auddie2015** [comment]
Just a thought... I did add the mod into an existing world and at the same time disabled OpenElevators. I can quickly recreate the world and see.

**Comment by Thutmose** [comment]
ya, see if it works with a new world

**Comment by auddie2015** [comment]
No. There are still no items showing. In NEI Subsets, the mod is blacked out and there are no items showing and it will not (edit) enable. It must be a conflict. I will mess around and turn stuff off I rarely use or only use one or two items from (Mekanism - only use the 7 burner smelters).

I will let you know after I tinker around. If you have an ideas in the meantime, they are welcome!

**Comment by Thutmose** [comment]
do things show up in creative tabs?

**Comment by auddie2015** [comment]
It's NEI. I have to use JEI to see them. I could try earlier versions of NEI but it is definitely NEI 1.12.2-4,1,238-universal. I Switched it in options to JEI and poof, there they were.

**Comment by auddie2015** [comment]
LOL This is after downloading every version of your core and elevators... I should have changed that first. But hey, I know now, right?

**Comment by Thutmose** [comment]
lol, good to know that is isn't entirely something on my end, is something showing up in logs anywhere?

**Comment by auddie2015** [comment]
Not that I could see. I searched for Thut and it was just going through OpenComputers injections and the mod and core themselves loading. 

I went through debug and the latest log. Just routine stuff. (Not that I am a programmer but I didn't see anything with ERROR in it. For yours or NEI. But JEI works so...
